### PR TITLE
Adding --skipInitProject to kfctl_test.jsonnet until CI quota is increased

### LINKED ISSUE #1562: presubmit build is failing with a quota error
See this [build](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/kubeflow_kubeflow/1535/kubeflow-presubmit/3366/)

The Error is:
```
ERROR: (gcloud.services.enable) RESOURCE_EXHAUSTED: Insufficient tokens for quota 'servicemanagement.googleapis.com/default_requests' and limit 'DefaultRequestsPe
rMinutePerProject' of service 'servicemanagement.googleapis.com' for consumer 'project_number:593963025935'.
```

### PR DESCRIPTION
**Pull request by ashahba**
fixes #1562

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1564)

### CODE REVIEW

**Comment by ashahba** [comment]
@jlewi did we want this in?

**Comment by jlewi** [comment]
@ashahba Apologies for the delay. (I also recommend doing /assign if you want someone to review your PRs) that way it shows up in (https://k8s-gubernator.appspot.com/pr) as needing attention.

I'll take a look

**Comment by jlewi** [comment]
I think this is a good fix; even when quota fixes are resolved.

I'll open up an issue to properly test the uninitialized project case.

**Comment by jlewi** [comment]
@ashahba sent you an invite to the org so you can trigger the tests on your PRs.

**Comment by ashahba** [comment]
@kunmingg Jeremy is right, looks like we are passing irrelevant options to `gcloud` in `kfctl.sh`.
I was a bit busy last couple of days, I'll take a closer look to see where we mix up arguments.
But by all means feel free to provide your feedback.

File: testing/workflows/components/kfctl_test.jsonnet [hunk 1]
```diff
@@ -156,6 +156,8 @@ local dagTemplates = [
         "--platform",
         "gcp",
         "--project",
+        // Temporary fix for https://github.com/kubeflow/kubeflow/issues/1562
+        "--skipInitProject",
```
Comment (kunmingg) [inline testing/workflows/components/kfctl_test.jsonnet]: line 160 and 161 should swap.

**Comment by ashahba** [comment]
@kunmingg and @jlewi turned out the while loop was making an extra `shift` near line https://github.com/kubeflow/kubeflow/pull/1564/files#diff-8a6f2d237f052b07ca594c20830ed9d2L111 
which ended up loosing some command line options.

But it's fixed now and this PR is ready for merge 🙂

**Comment by jlewi** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**